molecule/python/tests/test_python.py

```diff
@@ -49,6 +50,8 @@ def test_packages(host, pkg):
             ["certifi", "urllib3", "cryptography", "cffi", "MarkupSafe"],
         ),
         ("/tools/mitm6/.venv", ["mitm6"]),
+        # There is no venv for sqlmap since it has no dependencies.
+        # ("/tools/sqlmap/.venv", []),
```

Comment:
I'm not sure if you also wanted to alphabetize these parameters by the directory name.

Reply:
Please see commit b9ad57e.